frontend/apps/pwa/src/components/FormPicker.tsx

```diff
@@ -1,34 +1,69 @@
-import { FC, useCallback } from 'react';
+import React, { FC, useCallback } from 'react';
 import { FormDefinition, FormType } from 'core-api-client';
+import { FieldErrors } from 'react-hook-form';
+import { ErrorMessage } from '@hookform/error-message';
 
 import { useDatabase, useForms } from '../app/hooks';
+import { ValidationForm } from '../reducers/Former/types';
+import { registeredValidation } from '../features/former/validation';
 
 export type FormPickerProps = {
   forms: FormDefinition[];
   formId: string | undefined;
   disabled?: boolean;
   setFormId: (formId: string) => void;
+  register: any;
```

Comment:
Does the library export a type we could use here?

Reply:
yes, but it lead to a lot of problems :sweat_smile: I'll try using it again